pages/projects.tsx

```diff
@@ -14,6 +16,7 @@ const Projects = () => {
       </Head>
       <Navbar active={'projects'} />
       <main className="flex flex-col w-full mt-20">
+        <Toaster />
```

Comment:
could you check if we can remove this?

Reply:
If we remove it, it will not work like what we want.
 So here is the thing while using this library we have to do 2 things mainly -
1. first of all place <Toaster/> as a component at the upper level of the page.
2. Next we can send any notification on that particular page with just toast. success('hello world') and any other also.
 without any of the one element, we can't send notifications to the page. 
So when we put it on all pages we can send notifications to all of those pages with the help of toast.success("login successful").

Reply:
thanks for explaining. 👍